RELEASE/scripts/autoscend/auto_combat.ash

```diff
@@ -957,7 +957,11 @@ string auto_combatHandler(int round, monster enemy, string text)
 		return useSkill($skill[Apprivoisez La Tortue], false);
 	}
 
-	if(!in_zelda() && canUse($skill[Gulp Latte]) && (get_property("_latteRefillsUsed").to_int() == 0) && !get_property("_latteDrinkUsed").to_boolean())
+	if((!in_zelda() || my_class() == $class[Vampyre]) &&	//paths that do not use MP
+	canUse($skill[Gulp Latte]) &&
+	my_mp() * 2 < my_maxmp()		//gulp latte restores 50% of your MP. do not waste it.
```

Comment:
gulp latte restores 50% HP and 50% MP.
We use skills to cheaply convert MP to HP so really we can ignore the HP and just use it for MP restore.

While we previously gulped latte only 1/day. we did so immediately without concerned for wasted restore. So most likely we completely wasted that 1 we used a day.

I added a check for MP values. now it will only gulp it if MP is under 50% which means no MP is wasted by the gulp. Well ok, in theory some MP might be wasted if we also have MP regen and happened to win the combat without spending any MP. but it would be overly complex to check and prone to errors. And also if we had such high MP regen then our MP would not have dipped under 50% anyways.